iothub/device/src/Transport/Http/HttpClientHelper.cs

```diff
@@ -209,10 +209,6 @@ private static void AddCustomHeaders(HttpRequestMessage requestMessage, IDiction
             {
                 throw new IotHubClientException(ex.Message, IotHubStatusCode.NetworkErrors, ex);
             }
-            catch (OperationCanceledException)
```

Comment:
catching and just throwing again is weird, so just not catching anymore

Reply:
As long as it doesn't get caught by the Exception catch block below.